ksql-engine/src/main/java/io/confluent/ksql/KsqlEngine.java

```diff
@@ -278,7 +278,8 @@ public QueryMetadata getQueryExecutionPlan(final Query query) throws Exception {
       final String statementString,
       final MetaStore tempMetaStore,
       final MetaStore tempMetaStoreForParser,
-      final Map<String, Object> overriddenProperties
+      final Map<String, Object> overriddenProperties,
+      final boolean enforceTopicExistence
```

Comment:
Why did we add this parameter? I only ever see false being passed for it.

Reply:
Yes, we don't need it here. Removed it.